site/content/docs/heartbeat-checks/_index.md

```diff
@@ -101,3 +101,130 @@ Selecting a single check result page from the check overview page will give a de
+
+As mentioned earlier, we recommend using the `-m` and `--retry` options to specify timeout and retries to reduce the risk of false alerts or blocking the script. The corresponding options for wget are `-t` for retries and `-T` for timeout.
+{{< tabs "Shell" >}}
+{{< tab "Curl" >}}
+```BASH
+# run_backup.sh
+ 
+curl -m 5 --retry 3 https://ping.checklyhq.com/f0e0b1d3-665d-49d0-8bf0-3e6504c3d372
+```
+{{< /tab >}}
+{{< tab "Wget" >}}
+```BASH
+# run_backup.sh
+
+wget -T 5 -t 3 https://ping.checklyhq.com/87c05896-3b7d-49ae-83ff-5e81323a54c4
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+The above curl example can also be used in the [Heroku Scheduler](https://devcenter.heroku.com/articles/scheduler):
+
+`run_backup.sh && curl -m 5 --retry 3 https://ping.checklyhq.com/f0e0b1d3-665d-49d0-8bf0-3e6504c3d372 > dev/null`
+
+And similarly for [Render cron jobs](https://render.com/docs/cronjobs):
+
+`run_backup.sh && curl -m 5 --retry 3 https://ping.checklyhq.com/f0e0b1d3-665d-49d0-8bf0-3e6504c3d372`
+
+### Kubernetes CronJob
+Here is an example of how to add the curl command from earlier to a Kubernetes CronJob.
+
+{{< tabs "Kubernetes" >}}
+{{< tab "Kubernetes" >}}
+```BASH
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: nightly
+  namespace: example
+spec:
+  schedule: "0 2 * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: curl
+            image: docker.io/curlimages/curl:latest
+            imagePullPolicy: IfNotPresent
+            command:
+            - sh
+            - -c
+            args:
+            - 'curl -m 5 --retry 3 https://ping.checklyhq.com/f0e0b1d3-665d-49d0-8bf0-3e6504c3d372;'
+          restartPolicy: OnFailure
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+### Node.js
+In these examples we are using the built in [https.get](https://nodejs.org/api/https.html#httpsgeturl-options-callback) option, and then [axios](https://axios-http.com/).
+{{< tabs "Node.js" >}}
+{{< tab "Node.js" >}}
+```JS
+const https = require("https");
+
+// Sample URL
+const url = "https://ping.checklyhq.com/87c05896-3b7d-49ae-83ff-5e81323a54c4";
+
+const options = {
+  timeout: 5000,
+};
+
+https.get(url, options, (res) => {
+  console.log("statusCode:", res.statusCode);
+
+  res.on('data', (data) => {
+    console.log("responseBody:", data);
+  });
+});
+
+```
+{{< /tab >}}
+{{< tab "Axios" >}}
+```JS
+const axios = require('axios');
+
+axios.get('https://ping.checklyhq.com/87c05896-3b7d-49ae-83ff-5e81323a54c4').then(resp => {
+    console.log(resp.data);
+})
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+
+### Python
+Using the python [requests](https://requests.readthedocs.io/en/latest/) library with a timeout set to 5 seconds.
+
+{{< tabs "Python" >}}
+{{< tab "Python" >}}
+```PYTHON
+import requests
+
+# Heartbeat URL
+url = ["https://ping.checklyhq.com/c3f5f5bb-6e46-431a-b7b1-35105450cddc"]
```

Comment:
Did you try the python example? I think this might not work as you're putting the string into an array ("`[]`") and then referring to it as if it were just a normal string a few lines down 🤔

Reply:
I did not - I stole it from our quickstart guide in app. I'll try it today & if it doesn't work we'll need to change it in both places. Thanks!

Reply:
@ndom91 : Looks like you were right, didn't work with the `[]` around the string. We should fix that in the quickstart example too.